### DISCUSSION TITLE
Why is the version of pyodide-build so much ahead of pyodide? For example, 0.28, 0.27 ?

### DISCUSSION THREAD

**Discussion by hzhangxyz** [pyodide/pyodide-build#23] [Q&A] [answered]
Why is the version of pyodide-build so much ahead of pyodide? For example, 0.28, 0.27 ?

**Comment by ryanking13** [comment, marked as answer]
We separated the versioning of Pyodide and pyodide-build from Pyodide 0.26.1 to support faster release of pyodide-build.

pyodide > 0.26.1 is compatible to Pyodide 0.26 and (hopefully) be compatible to newer Pyodide versions.

You can check the compatibility table for Pyodide and pyodide-build in this file: https://github.com/pyodide/pyodide/blob/main/pyodide-cross-build-environments.json
